src/clj_icat_direct/queries.clj

```diff
@@ -248,12 +254,11 @@
        (if include-count? ",
                COUNT(*) OVER () AS total_count" "")
          " FROM r_coll_main AS c
-            JOIN r_objt_metamap AS om ON om.object_id = c.coll_id
-            JOIN r_meta_main AS m ON m.meta_id = om.meta_id
+            JOIN coll_avus AS ca ON ca.object_id = c.coll_id
```

Comment:
For `mk-files-in-folder` above, we pass in the name of the relevant AVUs CTE as an argument to the function (`avus-cte`) and use that in the join; I think I'd prefer if we did that here too, and it'd also enable combining the file and collection AVU CTEs as mentioned below.